src/bin/io/args.rs

```diff
@@ -48,6 +48,14 @@ pub(crate) struct WalletArgs {
     #[clap(long, action)]
     pub wait_for_tx: Option<bool>,
 
+    /// Output log level
+    #[clap(long)]
+    pub log_level: Option<String>,
```

Comment:
Consider using enums for log-level instead

https://github.com/clap-rs/clap/blob/495e49e1a70989f4c8904c355f90d6149f673ce2/examples/cargo-example-derive.rs#L3-L8